packages/py/python-aiohappyeyeballs/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : python-aiohappyeyeballs
+version    : 2.4.3
+release    : 1
+source     :
+    - https://github.com/aio-libs/aiohappyeyeballs/archive/refs/tags/v2.4.3.tar.gz : 0bb4dacfcedb8ef80f8983d4f66242b926c4c29dd271d7f8178f9517ea66abf6
```

Comment:
Please use this source from pypi:
`- https://files.pythonhosted.org/packages/source/a/aiohappyeyeballs/aiohappyeyeballs-2.4.4.tar.gz : 5fdd7d87889c63183afc18ce9271f9b0a7d32c2303e394468dd45d514a757745`
This will fix version detection as you can see in `pspec_x86_64.xml` file.
```
<Path fileType="library">/usr/lib/python3.11/site-packages/aiohappyeyeballs-0.0.0-py3.11.egg-info/PKG-INFO</Path>
```